openwisp_monitoring/check/tests/test_ping.py

```diff
@@ -110,8 +110,8 @@ def test_device_deleted(self):
         check.save()
         # dev = d
         # import ipdb; ipdb.set_trace()
```

Comment:
could you remove these comments please?

Reply:
Since, the entire test has been removed as per your above suggestion, it's been automatically removed :sweat_smile: